processing/src/main/java/io/druid/segment/filter/ExtractionFilter.java

```diff
@@ -54,15 +56,20 @@ public ExtractionFilter(
   {
     final Indexed<String> allDimVals = selector.getDimensionValues(dimension);
     final List<Filter> filters = Lists.newArrayList();
-    if (allDimVals != null) {
-      for (int i = 0; i < allDimVals.size(); i++) {
+    if (allDimVals != null)
+    {
```

Comment:
old code looked ok, I think druid codebase style is to keep parenthesis on the same line except for class/method opening.